src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { resolve, join } from 'path';
+import { readJson } from 'fs-extra';
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import remoteTags from 'remote-git-tags';
+
+const REQUIRED_TAG_FORMAT = '$version';
+
+// eslint-disable-next-line camelcase
+const cloneUrl: string = github.context.payload.repository?.clone_url;
+const tagFormat: string = core.getInput('tag-format');
```

Comment:
Should we be specifying the type for everything?

Reply:
It is good to explicit, but with libraries that have types like the actions toolkit, they will also be picked up automatically.

Reply:
So for this case, is it not necessary? ESLint has removed the one where it was inferred such as when the value is explicitly a string already but I wasn't sure for this one. I'd prefer to not add it if it's unnecessary.